indico/modules/events/editing/client/js/actions.js

```diff
@@ -0,0 +1,41 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import editableDetailsURL from 'indico-url:event_editing.api_editable';
+import getReviewURL from 'indico-url:event_editing.api_review_editable';
+
+import {indicoAxios} from 'indico/utils/axios';
+import {ajaxAction, submitFormAction} from 'indico/utils/redux';
+import {camelizeKeys} from 'indico/utils/case';
+
+export const SET_LOADING = 'SET_LOADING';
+export const SET_DETAILS = 'SET_DETAILS';
+
+export function loadTimeline(eventId, contributionId, type) {
+  const url = editableDetailsURL({confId: eventId, contrib_id: contributionId, type});
+  return ajaxAction(
+    async () => {
+      const result = await indicoAxios.get(url);
+      return camelizeKeys(result);
```

Comment:
Does this actually work? `result` is a Response and not the actual data.
For camelizing data, using the 5th (transformData) arg of ajaxAction would be better